### PR TITLE
bump ocsdk

### PR DESCRIPTION
**Pull request by elopezanaya**
# PR Details

### Id of the task, bug, story or other reference

 4255474

### Description
_Include a description of the problem to be solved_
The chat wi.dget remains in a loading state continuously

This is due to sometimes getchattoken endpoint is unresponsive, causing the widget to hand due to a improper way to send reject the promisse in ocsdk

## Solution Proposed

_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

patch was already applied in OCSDK

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

## Timeout getchatoken
image

## normal chat
image

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

**Review by xTEddie — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 ## Changed
 - core service url regex to include crmtest
+- Uptake [@microsoft/ocsdk@0.4.6](https://www.npmjs.com/package/@microsoft/ocsdk/v/0.4.6)
```
Comment (matan-hershberg-nuance) [inline CHANGELOG.md]:
```suggestion
- Uptake [@microsoft/ocsdk@0.5.6](https://www.npmjs.com/package/@microsoft/ocsdk/v/0.5.6)
```

**Review by matan-hershberg-nuance — APPROVED** [review]
